src/locales/en-US/errors.json

```diff
@@ -38,5 +38,13 @@
   "needSmallerString": "You have to give me a sentence smaller than {{number}} characters.",
   "invalidURL": "You have to give me a valid URL!",
   "invalidImageLink": "You have to give me a valid image link.",
-  "imageParsingError": "An error occurred while parsing the image."
+  "imageParsingError": "An error occurred while parsing the image.",
+  "invalidChannelType": "This command doesn't work in a {{type}} channel.",
```

Comment:
This is not being used anywhere

Reply:
imagineParsingError is not me, invalidChannelType is being used on line 29 of ChannelParameter.js.
https://github.com/SwitchbladeBot/switchblade/blob/dev/src/structures/command/parameters/types/ChannelParameter.js#L29